app/controllers/concerns/hydra/collections_controller_behavior.rb

```diff
@@ -142,6 +142,7 @@ def collection_params
 
     # Queries Solr for members of the collection.
     # Populates @response and @member_docs similar to Blacklight Catalog#index populating @response and @documents
+    # TODO: this should be moved into Hydra::Collections::SearchBehaviors
```

Comment:
I flatly disagree with this statement. Methods that are in SearchBuilder shouldn't be called directly from the controller.